reviews/utils/review_actions.py

```diff
@@ -210,3 +212,67 @@ def action_url(self):
     def description(self):
 
         return _('Verander aangestelde commissieleden')
+    
+class SendConfirmation(ReviewAction):
+
+    def is_available(self):
+
+        user = self.user
+        review = self.review
+
+        '''Only the secretary is able to send the confirmation letter and/or change the date.
+        The review needs to be closed and have an approved status.'''
```

Comment:
This should be moved to right after `def is_available(self):`, as it's a docstring.